module-2/backend/week_one.md

```diff
@@ -4,31 +4,90 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-1. List the five common HTTP verbs and what the purpose is of each verb.
+1. List the five common HTTP verbs and what the purpose is of each verb.<br>
+GET - Gets what you are asking it to<br>
+POST - Creates a new thing of what you asked it to<br>
+DELETE - Deletes what you ask it to<br>
+PATCH - Replaces part of what you are asking it to<br>
+PUT - Replaces ALL of what you are asking it to<br>
+
 2. What is Sinatra?
+A framework for creating web applications
+
 4. What is MVC?
+Model, Views, Controller
+A structure for how you view the pages, how the pages talk to eachother and how the database is manipulated.
+
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+So we always have a basic framework to work off of every time we start.
+
 6. What types of variables are accessible in our view templates without explicitly passing them?
+Instance variables
+
 7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
   
   ```ruby
   get '/horses' do
+  @count = 1
     erb :index
   end
   ```
 
 8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed` to the view?
+
+```ruby
+  get '/horses' do
+  :locals {:mr.ed => mr.ed}
+  end
+  ```
+ 
 9. What's the purpose of ERB?
+So you can render ruby in HTML
+
 10. Why do I need a development AND test database?
+Development you can change as you go. Test would be like a fixture that you can test for specific things.

```

Comment:
I wouldn't consider the test database a fixture exactly. Fixtures create data that we can use in our tests. Our test database is a database that exists separate from production/dev that allows us to create/destroy data that we can test without impacting those databases.